framework/src/postprocessors/InterfaceIntegralPostprocessor.C

```diff
@@ -0,0 +1,64 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "InterfaceIntegralPostprocessor.h"
+
+#include "libmesh/quadrature.h"
+
+template <>
+InputParameters
+validParams<InterfaceIntegralPostprocessor>()
+{
+  InputParameters params = validParams<InterfacePostprocessor>();
+  params.addClassDescription(
+      "Postprocessor computing the intgral of a quantity on an interface. Different type of "
```

Comment:
intgral -> integral
Different type -> Different types
remove one space after 'overriding'

Reply:
It looks like this wasn't actually resolved